benchmarks/CT18_bench.py

```diff
@@ -46,13 +46,13 @@ def benchmark_nnlo(self, Q0=1.295, Q2grid=(1e4,)):
         operator_card = {"Q2grid": list(Q2grid)}
         self.skip_pdfs = lambda _theory: [
             -6,
-            6,
-            22,
-            "ph",
-            "T35",
-            "V35",
+            # 6,
+            # 22,
+            # "ph",
+            "Tu8",
+            "Vu8",
         ]
-        self.run([theory_card], [operator_card], ["CT18NNLO"])
+        self.run([theory_card], [operator_card], ["NNPDF31_nnlo_as_0118_luxqed"])
```

Comment:
Keep the CT18 benchmark for CT18 - we also have the NNPDF benchmark with the default settings used there available ...

Reply:
Yes sorry I had changed it for running a bench but then I forgot to change it and I pushed it